common/app/Nav/Nav.jsx

```diff
@@ -217,6 +217,11 @@ export class FCCNav extends React.Component {
                 /> :
                 null
             }
+            { !shouldShowMapButton &&            
+                <li 
```

Comment:
```diff
-                <li 
+                <li
```

whitespace

Reply:
@BerkeleyTrue is it conventional for trailing spaces to not be allowed? The lint fails when there's even one trailing space. Does this impact performance or maintainability in any relevant way, because I think this is likely to unnecessarily annoy potential contributors.